src/main/java/boot/MetaModel.java

```diff
@@ -0,0 +1,48 @@
+package boot;
+
+import persistence.proxy.ProxyFactory;
+import persistence.sql.common.util.CamelToSnakeConverter;
+import persistence.sql.context.EntityPersister;
+import persistence.sql.context.impl.DefaultEntityPersister;
+import persistence.sql.dml.Database;
+import persistence.sql.dml.MetadataLoader;
+import persistence.sql.loader.EntityLoader;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class MetaModel {
+    private final Map<Class<?>, EntityPersister> entityPersisterMap = new HashMap<>();
+    private final Map<Class<?>, EntityLoader<?>> entityLoaderMap = new HashMap<>();
+
+    public void init(Metadata metadata, ProxyFactory proxyFactory) {
+        Map<Class<?>, MetadataLoader<?>> entityBindingMap = metadata.getEntityBinding();
+        Database database = metadata.database();
+
+        for (Map.Entry<Class<?>, MetadataLoader<?>> entry : entityBindingMap.entrySet()) {
+            Class<?> clazz = entry.getKey();
+            MetadataLoader<?> metadataLoader = entry.getValue();
+            entityPersisterMap.put(clazz,
+                    new DefaultEntityPersister(database, CamelToSnakeConverter.getInstance(), metadataLoader));
+            entityLoaderMap.put(clazz,
+                    new EntityLoader<>(database, metadataLoader, CamelToSnakeConverter.getInstance(), proxyFactory));
+        }
+    }
+
+    public EntityPersister entityPersister(Class<?> entityClass) {
+        if (entityClass == null || !entityPersisterMap.containsKey(entityClass)) {
+            throw new IllegalArgumentException("Not Found EntityPersister");
+        }
+
+        return entityPersisterMap.get(entityClass);
```

Comment:
```suggestion
        EntityPersister entityPersister = entityPersisterMap.get(entityClass);
        if (entityPersister == null) {
            throw new IllegalArgumentException("Not Found EntityPersister");
        }
        return entityPersister;
```
사실 get 메소드의 구현체가 key가 null이거나 값이 없으면 null을 반환하기 때문에 이렇게 표현하여도 되긴합니다